### PR TITLE
fix/update to fetch holiday-data up to 2000 results

### LINKED ISSUE #4: data/taiwant-holiday-cache.json was expired
[data/taiwant-holiday-cache.json](https://github.com/ensky/taiwan-holiday.js/blob/efdb362210e97cc16fa6c590f124388b266e7d2e/data/taiwan-holiday-cache.json) was expired and missing the holiday data of the year 2021, which will cause unexpected result if we do not provide our cache directory in `temp_dir`, here is the reproduce:
```js
const holiday = require('taiwan-holiday');

holiday.init(/* do not provide the temp_dir manually */)
	.then(() => {
		holiday.isHoliday('2021/1/1'); // expect true, but return false
	});
```

### PR DESCRIPTION
**Pull request by inkybr**
PR #5 doesn't provide the data of whole year of 2021. This PR attempts to add rest of them and fix #4 .

Data source is changed to the Open Data API of https://data.taipei . Because the api has a limit of 1000 result, it will be capped at 2021/4/23. So a change needs to made to parallelly fetch different segments of data and merge them to retrieve a complete set of holiday data.

For now, it only retrieves the first 2000 results. The api has only 1077 results as for now, so I figure 2000 may be enough until a more refined solution is made.

### CODE REVIEW

**Comment by bcgodev** [comment]
This PR is related to another PR #5 and issue #4

File: lib/opendata-tw.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ class OpendataTW {
 };
 
 // TODO: fetch method rather than url
-OpendataTW.url = 'https://data.ntpc.gov.tw/api/datasets/308DCD75-6434-45BC-A95F-584DA4FED251/json?size=2000';
+OpendataTW.urlFirst1000 = 'https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=0';
+OpendataTW.urlSecond1000 = "https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=1000";
```
Comment (ensky) [inline lib/opendata-tw.js]:
```suggestion
OpendataTW.urlSecond1000 = 'https://data.taipei/api/v1/dataset/b0011e96-3fc3-43ec-8bf5-07fb46dd22bb?scope=resourceAquire&q=&limit=1000=&offset=1000';
```

for quote consistency

**Review by ensky — COMMENTED** [review]
Thanks for the PR, nice work!

However, if 1000 results roughly means ~8 years of holidays, I think maybe we should check if there's another pages exists. Otherwise this library might needs to change at roughly 2029.

Check with result.count maybe a good idea?

File: lib/opendata-tw.js (hunk 1 above)
Comment (inkybr) [reply lib/opendata-tw.js]: Great suggestion. I should notice the code style consistency and readability. I'll make another patch soon.

Comment (inkybr) [reply lib/opendata-tw.js]: I've added a patch to implement the suggestion. Should we continue the discussion of fetching method in issue #9 and implement it in other PR?

**Pull request merged**